bin/test-wordpress-coding-standards.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
On a personal project I've used a composer script to trigger code sniffing: https://github.com/stevector/nerdologues-d8/blob/master/composer.json#L78